src/data/polynomial/erase_lead.lean

```diff
@@ -4,6 +4,7 @@ Released under Apache 2.0 license as described in the file LICENSE.
 Authors: Damiano Testa
 -/
 import data.polynomial.degree.basic
+import data.polynomial.degree
```

Comment:
I think we usually try to avoid `default.lean` imports to keep the import graph simpler; can you minimize this import?

Reply:
Hmm, not sure what to do with this, I'm trying to import something from the file `data/polynomial/degree.lean`, which is not in the folder `data/polynomial/degree`...

Reply:
Oh, weird. Hmm, we should probably rename that at some point, but it doesn't have to hold up this PR.